index_release.go

```diff
@@ -0,0 +1,117 @@
+package versionbundle
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/giantswarm/microerror"
+)
+
+const indexReleaseTimestampFormat = "2006-01-02T15:04:05.00Z"
+
+type IndexRelease struct {
+	Active      bool        `yaml:"active"`
+	Authorities []Authority `yaml:"authorities"`
+	Date        time.Time   `yaml:"date"`
+	Version     string      `yaml:"version"`
+}
+
+// TODO define and implement validation rules
+func ValidateIndexReleases(indexReleases []IndexRelease) error {
+	if len(indexReleases) == 0 {
+		return nil
```

Comment:
Function looks very clean and straightforward. Just to understand the big picture: `ValidateIndexReleases` will be called by the provider-operators?

Reply:
We will use the validation in `opsctl` and will then use `opsctl` in the `installation` repo's CI.